frontend/js/tests/explore/huesound/HueSound.test.tsx

```diff
@@ -0,0 +1,105 @@
+/* eslint-disable jest/no-disabled-tests */
+import { screen, within } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { HttpResponse, http } from "msw";
+import { SetupServerApi, setupServer } from "msw/node";
+import * as React from "react";
+import HueSound from "../../../src/explore/huesound/HueSound";
+import {
+  renderWithProviders,
+  textContentMatcher,
+} from "../../test-utils/rtl-test-utils";
+
+const release: ColorReleaseItem = {
+  artist_name: "Rorcal & Music For The Space",
+  caa_id: 19203085767,
+  color: [0, 0, 0],
+  dist: 0,
+  recordings: [
+    {
+      listened_at: 0,
+      track_metadata: {
+        additional_info: {
+          artist_mbids: [
+            "27030b71-0eb7-4dc9-93f5-6f43b9970d73",
+            "015ef401-99cf-4d84-a8b7-ccb141067c53",
+          ],
+          recording_mbid: "41abe69a-4f82-4b48-ac56-4a43de2b57d9",
+          release_mbid: "18f6bbb8-16b3-4e0d-823d-dc2c13ee9bed",
+        },
+        artist_name: "Rorcal & Music For The Space",
+        release_name: "Prelude to Heliogabalus",
+        track_name: "Prelude to Heliogabalus",
+      },
+    },
+  ],
+  release_mbid: "18f6bbb8-16b3-4e0d-823d-dc2c13ee9bed",
+  release_name: "Prelude to Heliogabalus",
+};
+jest.unmock("react-toastify");
+
+const user = userEvent.setup();
+
+describe("HueSound", () => {
+  let server: SetupServerApi;
+  beforeAll(async () => {
+    // Mock the server responses
+    const handlers = [
+      http.get("/1/explore/color/000000", () =>
+        HttpResponse.json({ payload: { releases: [release] } })
+      ),
+    ];
```

Comment:
Again, I don't believe that it should call hex `000000` on mount, so the test in this case needs to be refactored accordingly